addon/components/listing.js

```diff
@@ -243,3 +290,38 @@ export default class ListingComponent extends Component {
     return sourceData;
   }
 }
+
+function ensureOrderTriplesForExistingData({
+  subForms,
+  formStore,
+  sourceGraph,
+}) {
+  let shouldAddOrderTriples = subForms.some((subForm) => {
```

Comment:
My assumption was that if any sub form doesn't have a `sh:order` non of them will, so we add new ones.

I didn't check for cases where _some_ subforms have an order, but not all of them. In that case I don't know what the best way forward is, but I don't think it will happen anyways.

Let me know if the logic needs to be changed.

Reply:
Maybe we shouldn't do any fallback logic here and assume that the backend does it for the older sub forms?